downloads/downloads-impl/src/main/java/com/duckduckgo/downloads/impl/DataUriParser.kt

```diff
@@ -26,15 +26,25 @@ import okio.ByteString.Companion.decodeBase64
 class DataUriParser @Inject constructor() {
 
     fun generate(url: String): ParseResult {
-        val result = MIME_TYPE_REGEX.find(url)
+        val offset = url.indexOf(',')
+        if (offset == -1) {
+            return ParseResult.Invalid
+        }
+
+        val data = url.substring(offset + 1)
+        if (data.isEmpty()) {
+            return ParseResult.Invalid
+        }
```

Comment:
I think the unit test that's already there catches the first `ParseResult.Invalid` case, should we not add second test for this case?